AnnoyancesFilter/sections/popups.txt

```diff
@@ -3725,7 +3725,7 @@ topspb.tv##.social-popup
 ruerovideos.tv##.inline_el
 rupornotube.me##.inline_el
 onlinemultfilmy.ru###add_favorite_tooltip
-radyopop.com###fancybox-wrap
+listenradio.jp,radyopop.com###fancybox-wrap
```

Comment:
Please move `listenradio.jp###fancybox-wrap` to the `Annoyances - mobile-app.txt`.